linchpin/hooks/action_managers/action_manager.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABCMeta, abstractmethod
```

Comment:
@herlo tried moving it into __init__.py , linchpin was unable to resolve the ActionManager class , so I had to keep it in a separate file.